xline/src/main.rs

```diff
@@ -267,6 +267,19 @@ struct ServerArgs {
 
 impl From<ServerArgs> for XlineServerConfig {
     fn from(args: ServerArgs) -> Self {
+        let (storage, curp_storage) = match args.storage_engine.as_str() {
+            "memory" => (StorageConfig::Memory, StorageConfig::Memory),
+            "rocksdb" => (
+                StorageConfig::RocksDB(args.data_dir.clone()),
+                StorageConfig::RocksDB(args.curp_dir.unwrap_or_else(|| {
+                    let mut path = args.data_dir;
+                    path.push("curp");
+                    path
+                })),
+            ),
+            &_ => unreachable!(),
```

Comment:
Please print some useful information here, users may pass in other string in the command line

Reply:
Done